### PR TITLE
Remove references to removed GLES2 backend

### PR DESCRIPTION
**Pull request by Calinou**
- Update FAQ to remove issues resolved in 4.0, and add more advice on resolving VRR flicker issues.

### CODE REVIEW

File: tutorials/2d/particle_systems_2d.rst [hunk 1]
```diff
@@ -21,16 +21,18 @@ parameters and then adding randomness to them.
...
-the GLES2 backend, CPUParticles2D is a CPU-driven option with near feature parity with
-GPUParticles2D, but lesser performance. While GPUParticles2D is configured via a
-:ref:`class_ParticleProcessMaterial` (and optionally with a custom shader), the matching options
-are provided via node properties in CPUParticles2D (with the exception of the trail settings).
-You can convert a GPUParticles2D node into a CPUParticles2D node by clicking on the node in the
-inspector, and selecting "Convert to CPUParticles2D" in the "Particles" menu of the toolbar.
+Godot provides two different nodes for 2D particles, :ref:`class_GPUParticles2D`
+and :ref:`class_CPUParticles2D`. GPUParticles2D is more advanced and uses the
+GPU to process particle effects, but that limits it to higher end graphics API,
+and in our case to the GLES3 renderer. CPUParticles2D is a CPU-driven option
```
Comment (clayjohn) [inline tutorials/2d/particle_systems_2d.rst:27]: We should remove the reference to the GLES3 renderer here. Instead we should include a line saying that CPUParticles may be preferable on low-end systems or games with GPU bottlenecks.

File: tutorials/3d/3d_rendering_limitations.rst [hunk 2]
```diff
@@ -45,11 +45,12 @@ There are two main ways to alleviate banding:
   fullscreen debanding shader as a post-processing effect and is very cheap.
   Fullscreen debanding is only supported when using the GLES3 or Vulkan renderers.
```
Comment (clayjohn) [inline tutorials/3d/3d_rendering_limitations.rst:46]: The reference to GLES3 here (and above) should be removed. The OpenGL renderer does not render in HDR and debanding isn't available yet

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 3]
```diff
@@ -170,8 +170,9 @@ shader, this value can be used as desired.
 +----------------------------------------+--------------------------------------------------------+
 | in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction.                          |
 +----------------------------------------+--------------------------------------------------------+
-| in bool **OUTPUT_IS_SRGB**             | ``true`` when calculations happen in sRGB color space  |
-|                                        | (``true`` in GLES2, ``false`` in GLES3).               |
+| in bool **OUTPUT_IS_SRGB**             | ``true`` when calculations happen in sRGB color space, |
+|                                        | ``false`` when calculations happen in linear color     |
+|                                        | space.                                                 |
```
Comment (clayjohn) [inline tutorials/shaders/shader_reference/spatial_shader.rst:175]: While we are at it, I would change these to "true when output is in sRGB color space (true in gl_compatibility renderer, false in RD-based renderers)."

**Review by clayjohn — COMMENTED** [review]
Great work! Just a few little comments for your consideration

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Great, merged 🎉

**Pull request merged**